K8s-deployment/K8s-cluster/addons/cluster-autoscaler/rancher/README.md

```diff
@@ -0,0 +1,69 @@
+# Cluster Autoscaler for Rancher with RKE2
+
+This cluster autoscaler for Rancher scales nodes in clusters which use RKE2
+provisioning (Rancher v2.6+). It uses a combination of the Rancher API and the
+underlying cluster-api types of RKE2.
+
+## Configuration
+
+The `cluster-autoscaler` for Rancher needs a configuration file to work by
+using `--cloud-config` parameter. Example configuration file - 
+[config.yaml](./config.yaml).
+
+### Permissions
+
+The Rancher server account provided in the `cloud-config` requires the
```

Comment:
will this work with these permissions/scopes and also can u detail the api key  creation?

Reply:
Updated README